### PR TITLE
Fix spelling errors.

### PR DESCRIPTION
**Pull request by dpvc**
This fixes my dopy-and-pasted spelling error in "protocol".  I thought I remembered Peter already doing that, but it seems not to be fixed, so I'm submitting this pull request to do so.

### CODE REVIEW

**Comment by pkra** [comment]
>  I thought I remembered Peter already doing that, 

That happened in the core -- https://github.com/mathjax/MathJax/pull/1527 :smile_cat:

File: extensions/explorer.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ MathJax.Hub.Register.StartupHook('Sre Ready', function() {
       if (MathJax.Menu) {
         var menu = MathJax.Menu.menu.FindId('Explorer');
         if (menu) {
-          var items = menu.submenu.items;
+          var items = (menu.submenu||menu.menu).items;
```
Comment (pkra) [inline extensions/explorer.js]: cc @zorkow (since I don't know the code)

Comment (dpvc) [inline extensions/explorer.js]: Sorry, that shouldn't have been there.  It was left over from my trying to get it to work in v2.5 (there was a name change in the menus in v2.6).

**Comment by dpvc** [comment]
> That happened in the core 

Thanks!  I thought I was losing my memory.

File: extensions/explorer.js (hunk 1 above)
Comment (dpvc) [inline extensions/explorer.js]: I've removed the change.

**Comment by pkra** [comment]
LGTM!

**Pull request merged**